lib/active_model_serializers/adapter/json_api.rb

```diff
@@ -43,14 +44,13 @@ def default_key_transform
 
       # {http://jsonapi.org/format/#crud Requests are transactional, i.e. success or failure}
       # {http://jsonapi.org/format/#document-top-level data and errors MUST NOT coexist in the same document.}
-      def serializable_hash(options = nil)
-        options ||= {}
+      def serializable_hash(_options = nil)
```

Comment:
Which options would they pass to `as_json` that could be used by the `JsonApi` adapter?

Do you mean we should add `instance_options.merge!(options)` here?